recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    tag: 1.1.0
```

Comment:
Could we try downloading a tarball from that tag? That is the preferred method of source acquisition at conda-forge.